packages/framework/tests/Unit/Model/Order/Processing/OrderProcessorMiddleware/SetDeliveryAddressByDeliveryAddressUuidMiddlewareTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrameworkBundle\Unit\Model\Order\Processing\OrderProcessorMiddleware;
+
+use Shopsys\FrameworkBundle\Model\Customer\Customer;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddress;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressFacade;
+use Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser;
+use Shopsys\FrameworkBundle\Model\Order\Processing\OrderProcessorMiddleware\SetDeliveryAddressByDeliveryAddressUuidMiddleware;
+use Tests\FrameworkBundle\Test\MiddlewareTestCase;
+
+class SetDeliveryAddressByDeliveryAddressUuidMiddlewareTest extends MiddlewareTestCase
+{
+    public function testTransportIsAdded(): void
+    {
+        $orderProcessingData = $this->createOrderProcessingData();
+
+        $deliveryAddressUuid = '30a07736-85aa-4407-8a85-4e722ed3e035';
```

Comment:
could you extract this into a class constant, please? It helps the test readability IMHO

Reply:
oh now I see the three occurrences has no relation to each other so I would vote for using three different uuids :slightly_smiling_face:

Reply:
three different uuids it is